certbot-apache/certbot_apache/tests/centos_test.py

```diff
@@ -0,0 +1,138 @@
+# pylint: disable=duplicate-code
```

Comment:
We can, and the line wasn't working because of pylint bug that didn't allow disabling of duplicate-code locally, only throgh global configuration, but I've worked around the need for that already.